man/deepsea-stage-5.7

```diff
@@ -0,0 +1,42 @@
+.TH deepsea 7
+.SH NAME
+DeepSea stage 5 \- a deeper insight
+.SH DESCRIPTION
+.B DeepSea Stage 5 (Removal)
+is a Salt orchestration that deploys the non-core services of ceph.
```

Comment:
is a Salt orchestration that removes services.

Separate comment: I think role may still be less confusing than service.  Since systemd enables/disables services.  Roles are greater since they include configuration files.